src/modules/Game.class.js

```diff
@@ -1,68 +1,144 @@
 'use strict';
+import Core from './Core.class';
+
+import { GAME_STATUSES, KEY_BINDS } from '../lib/constants';
+
+class Game extends Core {
+  #hydrate() {
+    this.controls.startBtn.addEventListener('click', this.start.bind(this));
```

Comment:
I like your structure and splitting logic 🔥 👍 

But using HTML elements in the Game class will always lead to tests failing in this task - FYI.